RHDEVDOCS-4410 - Doc: Resource type updates-Import from Git & deploy image (#51733)
openshift/openshift-docs: modules/odc-deploying-java-applications.adoc

. In the optional *Application Name* field, enter a unique name for your application for the resource labelling.

> . In the *Name* field, enter a unique component to name the associated resources.

Reviewer: ```suggestion
. In the *Name* field, enter a unique component name for the associated resources.
```
This phrasing does not make sense to me because it's saying that the "component" rather than the user is naming resources, and I don't think this meaning is what you're trying to express. Please use my suggested phrasing or some other alternative phrasing to get across the idea that the user is assigning a component name to a resources.

Author: Good catch. Thank you!

> . In the *Name* field, enter a unique component name to name the associated resources.

Reviewer 2: **Suggestion**: The word "name" has been used too many times in a relatively short sentence. Consider restructuring the sentence for better readability and aesthetics. For example:

In the **Name** field, enter a unique component name to indicate the associated resources.

OR

In the **Name** field, enter a unique component to name the associated resources.

----------------------------------
Another generic advice would be to restructure such sentences in the following format:

_To <achieve some objective>, <perform some action>._

I understand that these bullet points follow other sentences that are beyond the scope of your PR, and hence the suggestion for the future.

> . Optional: You can select a different resource type from the default resource type if needed from the *Advanced options* -> *Resource type* drop-down list. 

Reviewer 2: Drop the "if needed". The "Optional:" tag at the start of the sentence implies that this instruction is applicable only if needed for a specific use case.

Suggested revision:

Optional: To select a different resource type from the list of default resource types, use the **Advanced options** -> **Resource type** drop-down list.

Author: To maintain consistency, I would like to go with the second suggested version as follows:

Optional: Using the *Advanced options* -> *Resource type* drop-down list, select a different resource type from the list of default resource types. 

Cc @divyanshiGupta @sanketpathak

> . Optional: For the *Advanced options* -> *Resource type*, you can set the default resource type from the *User Preferences* page. Also, you can modify the default selected resource type by using the *Resource type* drop-down list. 

Reviewer 3: ```suggestion
. Optional: You can select a different resource type from the default resource type if needed from the Advanced options -> Resource type drop-down list.
```

Reviewer 3: @smunje1 selecting a default resource type from user preferences page should already be there in the user-preferences doc. Can you check and confirm? If not then we can include that here.

Author: @divyanshiGupta Yes, you are right. It seems we have already covered selecting a default resource type from the user preferences page as part of RHDEVDOCS-4236 https://github.com/openshift/openshift-docs/pull/49691

> . In the *Advanced options*, click on *Create a Route to the Application* to configure a public URL for your deployed application.

Reviewer 2: The previously stated generic suggestion of using the "To achieve an objective, perform an action" format applies here also. But I don't expect you to make sweeping changes to the whole document that's simply beyond the scope of the current PR. 

However, drop the "on" from this sentence to adhere to minimalism principles. For example:

In the *Advanced options*, click *Create a Route to the Application* to configure a public URL for your deployed application.
